docs/intelligentapps/finetune.md

```diff
@@ -1,317 +1,403 @@
+- `train.json` – Used for training the adapter.
+- `test.json` – Used for evaluating the adapter's performance during and after training.
 
-The results of the fine-tuning will be stored in the Azure Files.
-To explore the output files in the Azure File share, you can navigate to the Azure portal using the link provided in the output panel. Alternatively, you can directly access the Azure portal and locate the storage account named `STORAGE_ACCOUNT_NAME` as defined in `./infra/fintuning.config.json` and the file share named `FILE_SHARE_NAME` as defined in `./infra/fintuning.config.json`.
+Both files must use the JSON format, where each line is a separate JSON object representing a single sample. Each sample should contain a list of messages exchanged between a user and an assistant.
 
-![file-share](./images/finetune/log-finetuning-files.png)
+Every message object requires two fields:
 
-### View logs
+- `content`: the text of the message.
+- `role`: either `"user"` or `"assistant"`, indicating the sender.
 
-Once the fine-tuning job has been started, you can access the system and console logs by visiting the [Azure portal](https://portal.azure.com).
+See the following examples:
 
-Alternatively, you can view the console logs directly in the VSCode Output panel.
+```json
+{"messages": [{"content": "Hello, how do I reset my password?", "role": "user"}, {"content": "To reset your password, go to the settings page and click 'Reset Password'.", "role": "assistant"}]}
 
-![log-button](./images/finetune/notification-finetune.png)
+{"messages": [{"content": "Can you help me find nearby restaurants?", "role": "user"}, {"content": "Sure! Here are some restaurants near your location: ...", "role": "assistant"}]}
 
-> [!NOTE]
-> The job might take a few minutes to initiate. If there is already a running job, the current one may be queued to start later.
+{"messages": [{"content": "What is the weather like today?", "role": "user"}, {"content": "Today's forecast is sunny with a high of 25°C.", "role": "assistant"}]}
+```
 
-#### View and query logs on Azure
+Training tips:
 
-After the fine-tuning job is triggered, you can view logs on Azure by selecting the "*Open Logs in Azure Portal*" button from the VSCode notification.
+- There is no comma needed at the end of each sample line.
+- Include as many high-quality and diverse examples as possible. For best results, collect at least a few thousand training samples in your `train.json` file.
+- The `test.json` file can be smaller, but should cover the types of interactions you expect your model to handle.
 
-Or, if you've already opened the Azure Portal, find job history from the "*Execution history*" panel to the Azure Container Apps job.
+Create `train.json` and `test.json` files with one JSON object per line, each containing a brief back-and-forth conversation between a user and an assistant. The quality and quantity of your data will greatly affect the effectiveness of your LoRA adapter.
```

Comment:
Should this also be a bulleted list item?

Reply:
> Should this also be a bulleted list item?

Good catch! Changed to a bulleted list item